src/main/webapp/app/shared/components/code-button/code-button.component.ts

```diff
@@ -433,4 +441,79 @@ export class CodeButtonComponent implements OnInit {
             return 'artemisApp.exerciseActions.cloneExerciseRepository';
         }
     }
+
+    private initTheia(profileInfo: ProfileInfo) {
+        if (profileInfo.activeProfiles?.includes(PROFILE_THEIA) && this.exercise()) {
+            let exercise: ProgrammingExercise = this.exercise()!;
```

Comment:
```suggestion
            let exercise = this.exercise()!;
```

Reply:
When typescript can infer the type, we generally avoid to mention the type, in particular if it's clear from the variable name